whipper/common/config.py

```diff
@@ -101,6 +101,10 @@ def getDefeatsCache(self, vendor, model, release):
         option = self._getDriveOption(vendor, model, release, 'defeats_cache')
         return option == 'True'
 
+    def getCdparanoia(self, vendor, model, release):
```

Comment:
```suggestion
    def getCdParanoia(self, vendor, model, release):
```

Same casing as other instances.